modules/nf-core/bakta/baktadbdownload/main.nf

```diff
@@ -0,0 +1,45 @@
+process BAKTA_BAKTADBDOWNLOAD {
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::bakta=1.6.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bakta:1.6.0--pyhdfd78af_0' :
+        'quay.io/biocontainers/bakta:1.6.0--pyhdfd78af_0' }"
+
```

Comment:
Interesting to see a module without input :)

Reply:
It always looks... 'dirty' to me :laughing;